tools/container_run_ci

```diff
@@ -28,26 +28,35 @@ set -o pipefail
 opts=$(getopt -o vht: --long verbose,help,target: -n 'parse-options' -- "$@") || usage 1
 eval set -- "$opts"
 while true; do
-  case "$1" in
-    -v | --verbose ) set -x; shift ;;
-    -h | --help ) usage 0 ;;
-    -t | --target ) target="$2"; shift 2 ;;
-    -- ) shift; break ;;
-    * ) break ;;
-  esac
+    case "$1" in
+    -v | --verbose)
+        set -x
+        shift
+        ;;
+    -h | --help) usage 0 ;;
+    -t | --target)
+        target="$2"
+        shift 2
+        ;;
+    --)
+        shift
+        break
+        ;;
+    *) break ;;
+    esac
```

Comment:
can we prevent this IMHO excessive line breaking here?

Reply:
I also don't like it, but I can't find an option for that.
https://github.com/mvdan/sh/blob/master/cmd/shfmt/shfmt.1.scd
```
[*.sh]
# like -i=4
indent_style = space
indent_size = 4

# --language-variant
shell_variant      = posix
binary_next_line   = true
# --case-indent
switch_case_indent = true
space_redirects    = true
keep_padding       = true
# --func-next-line
function_next_line = true
```
But I can live with it